app/controllers/api/edge.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+use Ahc\Jwt\JWT;
+use Ahc\Jwt\JWTException;
+use Appwrite\DSN\DSN;
+use Appwrite\Extend\Exception;
+use Appwrite\URL\URL as AppwriteURL;
+use Appwrite\Utopia\Request;
+use Appwrite\Utopia\Response;
+use Utopia\App;
+use Utopia\CLI\Console;
+use Utopia\Database\Document;
+use Utopia\Queue\Client as SyncIn;
+use Utopia\Queue\Connection\Redis as QueueRedis;
+use Utopia\Validator\ArrayList;
+use Utopia\Validator\Text;
+
+App::init()
+    ->groups(['edge'])
+    ->inject('request')
+    ->action(function (Request $request) {
+
+        $token = $request->getHeader('authorization');
+        $token = str_replace(["Bearer"," "], "", $token);
```

Comment:
Aren't we missing here a space that needs to be removed?
`Authorization: Bearer {token}`